### DISCUSSION TITLE
Ddlc edit

### DISCUSSION THREAD

**Discussion by niapng** [Monika-After-Story/MonikaModDev#7839] [general - submod / external content]
Hi! Today i made a ddlc background for phones, everything with official art so i thought of sharing it here!

![ddlc background ](https://user-images.githubusercontent.com/73759533/121956177-17d09700-cd61-11eb-87b1-d154e3ef934e.png)

**Comment by niapng** [comment]
I know it looks bad but on phones looks better!

**Comment by Retrolovania** [comment]
Interesting...
(I think the official Fan Pack has mobile backgrounds, though.)

**Comment by niapng** [comment]
It does, but i prefered this version more accurate as the main menu un the original game!
